Makefile

```diff
@@ -95,9 +108,22 @@ vet:
 	go vet ./...
 
 # Generate code
-generate: controller-gen
+generate: controller-gen types
 	$(CONTROLLER_GEN) object:headerFile=./hack/boilerplate.go.txt paths=./api/...
 
+# generates many other files (listers, informers, client etc).
+api/addon/v1alpha1/zz_generated.deepcopy.go: $(TYPES)
+	ln -s v1
```

Comment:
What's this symlink for?

Reply:
The symlink v1 is just a temp directory to verify the update of api/addon/v1alpha1/zz_generated.deepcopy.go file